text/rustconf-steering-committee.md

```diff
@@ -5,46 +5,49 @@
+- "CFP": Stands for "Call for Proposals". This is the period and process during which we will solicit talk submissions for RustConf. The Program Committee will select submissions from the CFP to appear on the program and might also advocate for inviting several speakers who did not submit a talk. 
+  
+# Proposal
 
-A group of ~5 volunteers from within the Leadership Council and the Project Directors responsible for providing input and direction to the Rust Foundation staff team regarding RustConf 2024. The Rust Foundation will lean on the steering committee to advocate for areas of desired involvement besides the traditional UnConference Day and Program Committee. 
+The Rust Foundation staff team is proposing the formation of a RustConf Steering Committee with the following parameters:
+- Members: 4 total members from the Rust Project -- one leadership council member, one Project Director member, and two members from the wider Rust community. We feel that an even split between leadership members and community members makes sense as the former positions can provide context and Project authority while the latter can advocate for those who are not necessarily in a position of power within the Project. We will lean on the Leadership Council as a whole to select this group based on a process of self-nomination initiated by the Foundation and promoted by the Leadership Council.  
+- Scope: The Foundation would like the Steering Committee's involvement in the following areas:
+ > - **Program Committee:** The Steering Committee should establish the structure, personnel, and processes of the Program Committee with oversight and collaboration from the Rust Foundation. The Foundation will take this plan from the Steering Committee to formally set up the Program Committee which will select talks after the closure of the call for talk proposals. Steering Committee members will not be on the Program Committee or be involved in selecting talks, although they can advise the Program Committee on key priorities as talks are being selected. To stay on schedule, the Foundation will likely need to open the CFP earlier than these two committees are in place, however, we will share our intentions for talk tracks/suggested topics with the Leadership Council in advance for feedback. 
+>  - **UnConf Day:** The Foundation would be pleased to offer an UnConf Day again in 2024, but would lean on the Steering Committee to decide this. If they decide there should be an UnConf Day after RustConf, the Foundation would like the Steering Committee to provide us with a basic plan including content and identification of people from the Project we should be logistically coordinating with ("UnConf Day Organizers") to ensure proper supplies and resources are available to UnConf organizers and attendees. The Foundation would welcome the Steering Committee's participation in planning and running the UnConf alongside the Organizers they identify.
```

Comment:
Again, this is written too much from the author's perspective. Try to avoid an us/them framing and talk about all groups in the third person. 
```suggestion
>  - **UnConf Day:** The Steering Committee is in charge of deciding whether to include an Unconf style event in the programming. If they decide there should be an UnConf Day after RustConf, the Steering Committee is required to provide the Foundation with a basic plan including content and identification of people from the Project that should be involved in coordinating logistics.
```